lms/static/sass/shared/_footer-edx.scss

```diff
@@ -321,6 +396,63 @@ footer#footer-edx-v3 {
   @include media-breakpoint-up(sm) {
     padding: $baseline $baseline*0.5;
 
+    .column-1 {
+      flex: 0 0 58.33333%;
+      max-width: 58.33333%;
+    }
+
+    .column-2 {
+      flex: 0 0 50%;
+      max-width: 50%;
+    }
+
+    .column-3 {
+      flex: 0 0 50%;
+      max-width: 50%;
+    }
```

Comment:
Please group selectors to keep the code cleaner, i.e. this is can be re-written;
```
.column-2,
.column-3 {
  flex: 0 0 50%;
  max-width: 50%;
}
```